Sources/Core/CharcoalViewController.swift

```diff
@@ -92,36 +92,37 @@ extension CharcoalViewController: FilterViewControllerDelegate {
         popToRootViewController(animated: true)
     }
 
-    func filterViewController(_ viewController: FilterViewController, didSelectFilter filterNode: CCFilterNode) {
-        guard !filterNode.isLeafNode else { return }
+    func filterViewController(_ viewController: FilterViewController, didSelectFilter filter: Filter) {
+        guard !filter.isLeafFilter else { return }
```

Comment:
Is there a more suitable name for this parameter? I don't feel like `isLeafFilter` works anymore. Maybe it is better to use `filter.subFilters.isEmpty`directly.

Reply:
I agree 😉

Reply:
Fixed now.